src/mscorlib/ref/mscorlib.cs

```diff
@@ -10276,6 +10276,7 @@ public sealed partial class RuntimeCompatibilityAttribute : System.Attribute
     }
     public static partial class RuntimeHelpers
     {
+        public static new bool Equals(object o1, object o2) { throw null; }    
```

Comment:
It is not necessary, but it does not hurt anything if you do it. This file should be deleted once proper mscorlib façade gets built in corefx.